### PR TITLE
Replace ontology by domain

### PR DESCRIPTION
**Pull request by NoB0**

### CODE REVIEW

File: tests/domain/test_domain.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"""Tests ofr MovieDomain class."""
```
Comment (WerLaj) [inline tests/domain/test_domain.py:1]: Nit: typo

Comment (NoB0) [reply tests/domain/test_domain.py:1]: Done.

File: tests/data/test_domain.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: "Test domain"
+
+slot_names:
+  genres: 
+    - system_requestable
```
[the remark this answers is not in the document]
Comment (NoB0) [reply tests/data/test_domain.yaml:5]: The structure is the same. In YAML, you can use `[object1, object2, ...]` or a sequence of `-` to represent a value of type list.

File: data/movies_domain.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+name: Movies
+
+slot_names:
+    genres: ["system_requestable", "user_requestable", "slots_annotation", "multiple_values"]
+    keywords: ["system_requestable", "slots_annotation"]
+    imdb_rating: ["user_requestable", "slots_not_required_NLU"]
```
[the remark this answers is not in the document]
Comment (NoB0) [reply data/movies_domain.yaml:6]: It is from the original codebase, I believe that it means that this slot is not annotated by the NLU component.

**Pull request merged**